ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -112,12 +112,19 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) =>
         content={<div><b>Warning:</b> {warning}</div>}
       />
     }
-    {genotype.numAlt >= 0 ?
+    {variant.svType ?
+      <Header.Content>
+        {Number.isInteger(genotype.numAlt) && genotype.numAlt >= 0 &&
+        <div>
+          {genotype.numAlt > 0 ? <b><i>X</i></b> : '-'}/{isHemiX || genotype.numAlt < 2 ? '-' : <b><i>X</i></b>}
+        </div>}
+        {Number.isInteger(genotype.cn) && genotype.cn !== (isHemiX ? 1 : 2) &&
```

Comment:
This is the wrong logic. If there is no genotype, like in the case of all the Exome SVs, then you need to always show CN. If there is a genotype, then you only show CN if its in some ways alt. I think you need a helper function for this:
```
const svGenotype = genotype => {
  const hasGenotype = Number.isInteger(genotype.numAlt) && genotype.numAlt >= 0
  const isAltCn = Number.isInteger(genotype.cn) && genotype.cn !== (isHemiX ? 1 : 2)
  if (!hasGenotype) {
    return <span>CN: {isAltCn ? <b><i>{genotype.cn}</i></b> : genotype.cn}</span>
  }
  genotypeDisplay = 
    <span>
      {genotype.numAlt > 0 ? <b><i>X</i></b> : '-'}/{isHemiX || genotype.numAlt < 2 ? '-' : <b><i>X</i></b>
    </span>
  return isAltCn ?  
    <span>
      {genotypeDisplay} <br /> CN: <b><i>{genotype.cn}</i></b>
    </span> : genotypeDisplay
}
```

Reply:
Is it possible to have any cases that have no genotype and no `genotype.cn`?

Reply:
yes, but you already handle that case before calling this helper: https://github.com/broadinstitute/seqr/pull/1882/files/93d4a461c6638f76499f9ab91f8ef3dbfdd20742#diff-812635e8be95aff2d4e3c613b762ba4688ec63ea3168bd90a7c4f5700ef7d00fR176